tensilelite/Tensile/Activation.py

```diff
@@ -457,15 +457,18 @@ def getExpModule(self, cDataType, vgprIn, vgprOut):
                                        sdwa=SDWAModifiers(dst_sel=select_bit, dst_unused=UnusedBit.UNUSED_PRESERVE, \
                                                           src0_sel=select_bit), \
                                        comment="exp step 2"))
-                    module.add(SNop(waitState=0, comment="1 wait states")) #workaround for emulator
+                    if globalParameters["ArchCaps"][globalParameters["CurrentISA"]]["TransOpWait"]:
+                        module.add(SNop(waitState=0, comment="1 wait states"))
```

Comment:
Please init the Caps in _initArchCaps() rather then check ISA version in kernel writer stage.
def _initArchCaps(isaVersion) -> dict:

Reply:
done